### PR TITLE
[Telegram] Improved sending to a particular chat section

### PR DESCRIPTION
**Pull request by FrancescoRisso**
Changed the README.md file to include more information:

Specified that the chatId in the APIs must be followed by an "L" or it won't work (at least, not for group chats).
Also mentioned that you can store the chatId inside a variable so that you can write it only once for the entire rules file.

### CODE REVIEW

File: bundles/org.openhab.binding.telegram/README.md [hunk 1]
```diff
@@ -155,12 +155,19 @@ These actions will send a message to all chat ids configured for this bot.
...
 telegramAction.sendTelegram(1234567L, "Hello world!")
 ```
 
+You can also define a global variable to use in all your rules file:
+
+```
+val Id = 1234567L
+telegramAction.sendTelegram(Id, "Hello world!")
+```
+
```
Comment (ZzetT) [inline bundles/org.openhab.binding.telegram/README.md:170]: Hm... Here I tend to say that defining a global variable instead of passing the actual value is something that is not related to the Telegram binding and applies to all other bindings that define an action where you can pass something. If somebody doesn't know about that, I would refer to the general rule guide of OH (https://www.openhab.org/docs/configuration/rules-dsl.html).

**Review by ZzetT — COMMENTED** [review]
Thanks for improving our documentation. I added my remarks to your two proposed changes.

File: bundles/org.openhab.binding.telegram/README.md (hunk 1 above)
Comment (FrancescoRisso) [reply bundles/org.openhab.binding.telegram/README.md]: Agree, removed

**Review by ZzetT — APPROVED** [review]
ok for me

**Comment by ZzetT** [comment]
Please make sure that your commits are signed off: https://github.com/openhab/openhab-addons/pull/8733/checks?check_run_id=1244340033

**Review by cpmeister — APPROVED** [review]
LGTM

**Comment by cpmeister** [comment]
@FrancescoRisso We are no longer currently accepting changes against the 2.5.x branch. We are in the process of putting all development efforts into OH3 which is on the `main` branch. If you want this to be merged into the `2.5.x` branch please make a PR on the `main` branch first. We are only allowing changes in the `main` branch to be backported to 2.5.x.

Sorry for the extra hassle, it was just unlucky timing. The PR procedure should return to normal in January after the official release of OH3.
